scripts/pylint_custom_plugin/tests/test_pylint_custom_plugins.py

```diff
@@ -2568,4 +2568,73 @@ def __init__(self):
                     msg_id="docstring-admonition-needs-newline", node=class_node
                 )
         ):
-            self.checker.visit_classdef(class_node)
\ No newline at end of file
+            self.checker.visit_classdef(class_node)        
+    
+    
+class TestCheckNoAliasGeneratedCode(pylint.testutils.CheckerTestCase):
+    CHECKER_CLASS = checker.CheckNoAliasGeneratedCode
+
+    def test_ignores_correct_alias_code(self):
+        module_node = astroid.extract_node(
+            """
+            import something as somethingElse
+            """
+        )
+
+        with self.assertNoMessages():
+            self.checker.visit_module(module_node)
+
+    def test_catches_incorrect_alias_code(self):
+        import_one = astroid.extract_node(
+            'import Something'
+           
+        )
+        import_two =  astroid.extract_node(
+            'import Something2 as SomethingTwo'
```

Comment:
I think we should have a `from something import something1 as something2` as a scenario in the tests